packages/next/src/build/swc/index.ts

```diff
@@ -25,6 +24,60 @@ const infoLog = (...args: any[]) => {
   Log.info(...args)
 }
 
+/**
+ * Based on napi-rs's target triples, returns triples that have corresponding next-swc binaries.
+ */
+const getSupportedArchTriples: () => Record<string, any> = () => {
+  const { darwin, win32, linux, freebsd, android } = platformArchTriples
+
+  return {
+    darwin,
+    win32,
+    linux: {
+      // linux[x64] includes `gnux32` abi, with x64 arch.
+      x64: linux.x64.filter((triple: {abi: string}) => triple.abi !== 'gnux32'),
+      arm64: linux.arm64,
+      // This target is being deprecated, however we keep it in `knownDefaultWasmFallbackTriples` for now
+      arm: linux.arm,
+    },
+    // Below targets are being deprecated, however we keep it in `knownDefaultWasmFallbackTriples` for now
+    freebsd: {
+      x64: freebsd.x64,
+    },
+    android: {
+      arm64: android.arm64,
+      arm: android.arm,
+    },
```

Comment:
We don't build these anymore, should they be listed here?

Reply:
Yes, we use those for wasm fallback binding check (https://github.com/vercel/next.js/blob/6c3560dbbd0f1b083ec9e15791cd94d1f113452e/packages/next/src/build/swc/index.ts#L145) for now. As commented, we can get rid of all at once later but for now keep it to not to break existing logics.